gsad/src/gsad_gmp.c

```diff
@@ -10253,7 +10252,7 @@ save_scanner_gmp (gvm_connection_t *connection, credentials_t *credentials,
                 "<credential id=\"%s\"/>"
                 "</modify_scanner>",
                 scanner_id, name, comment ?: "", host, port, type,
-                strcmp (which_cert, "new") == 0 ? ca_pub : "", credential_id);
+                strcmp (which_cert, "new") == 0 ? ca_pub : "", credential_id ? credential_id : "");
```

Comment:
To fix running the CI it just needs some reformatting. This should help
```suggestion
                strcmp (which_cert, "new") == 0 ? ca_pub : "",
                credential_id ? credential_id : "");
```